boards/arm64/rpi_5/Kconfig.defconfig

```diff
@@ -17,3 +17,10 @@ config XEN_DOM0
 	default y
 
 endif # BOARD_RPI_5_XEN_DOM0
+
+if BOARD_RPI_5_XEN_DOMD
```

Comment:
This is also not okay to made separate board for this. You may use custom sample/application, please check other applications and how they override device tree and Kconfigs.

Reply:
Huh. Sorry, but it will stay here for RPI5 experiments and exercise demonstrations as I can't even imagine how to resolve DT dependencies between clean RPI5, rpi5 dom0, rpi5 domd (uses at least one hw IP) and xenvm. Lets concider samples/basic/blinky:
 - adding rpi_5.overlay for clean RPI5 will work and might work with dom0 snippet, but will not work for "domD".
 - adding RPI5 specific xenvm.overlay will enable support for RPI5 DomD, bit it can't be done for obvious reasons as xenvm is generic.

Reply:
removed and replaced with snippet